### PR TITLE
feat(h13): Add BlurHash to Image

### PR DESCRIPTION
**Pull request by MrSltun**
This PR resolves [] 

> [!NOTE]
> This PR is part of [#hack13-blurhash](https://artsy.slack.com/archives/C06E4ND6EBV) 🔒

### Description

This PR adds BlurHash to the Image component

### Videos
| Android | iOS |
|---|---|
| <video src="https://github.com/artsy/palette-mobile/assets/20655703/96198b3c-bfcf-45e9-9de3-27b72380d470" /> | <video src="https://github.com/artsy/palette-mobile/assets/20655703/df8d077a-5021-4b95-b8c8-ff355e353402" /> |

<details>
  <summary>📦 Published PR as canary version: <code>13.1.10--canary.173.1364.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @artsy/palette-mobile@13.1.10--canary.173.1364.0
  # or 
  yarn add @artsy/palette-mobile@13.1.10--canary.173.1364.0
  ```
</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -147,4 +149,4 @@
       "prettier --write --ignore-unknown"
     ]
   }
-}
+}
```
Comment (gkartalis) [inline package.json:152]:
```suggestion
}

```

File: src/elements/Image/Image.stories.tsx [hunk 2]
```diff
@@ -62,3 +62,34 @@ storiesOf("Image", module)
...
+    />
+  ))
+  .add("Without automatic resizing + blurhash", () => (
+    <Image
+      blurhash="LXK,+3H#yArKH~bGx:tM.atxM*k9"
+      aspectRatio={0.62}
+      performResize={false}
+      src="https://d32dm0rphc51dk.cloudfront.net/A983VUIZusVBKy420xP3ow/normalized.jpg"
+    />
+  ))
```
Comment (gkartalis) [inline src/elements/Image/Image.stories.tsx:95]: ❤️

**Review by gkartalis — COMMENTED** [review]
Looks awesome! 

One thought only:

- not sure why the flags were added here `ios/PaletteMobile.xcodeproj/project.pbxproj` we should check if the are actually needed or if not lets get rid of them.

File: src/elements/Image/Image.tsx [hunk 3]
```diff
@@ -24,6 +25,8 @@ export interface ImageProps extends CustomFastImageProps {
   src: string
   /** Gemini resize_to param  */
   geminiResizeMode?: GeminiResizeMode
+  /** BlurHash code */
```
Comment (anandaroop) [inline src/elements/Image/Image.tsx:28]: **suggestion:** more descriptive comment for Intellisense…

```suggestion
  /** Compact textual representation of image for placeholder */
```

**Review by anandaroop — APPROVED** [review]
Sweet!

I'll defer to the Eigen experts, just one lil suggestion from me.

**Comment by gkartalis** [comment]
Turns out `ios/PaletteMobile.xcodeproj/project.pbxproj` is supposed to be there will add it on another pr

**Review by damassi — APPROVED** [review]
Niceee

**Comment by artsyit** [comment]
:rocket: PR was released in [`v13.1.11`](https://github.com/artsy/palette-mobile/releases/tag/v13.1.11) :rocket:

**Pull request merged**